src/index.html

```diff
@@ -17,11 +30,354 @@
     />
     <link
       rel="stylesheet"
-      href="styles/index.scss"
+      href="styles/styles.scss"
```

Comment:
The stylesheet is linked with a `.scss` file extension. Browsers do not natively support SCSS files, so please ensure that the SCSS is compiled to CSS. This is crucial for the functionality of the webpage.